.github/workflows/e2e_hardware.yml

```diff
@@ -0,0 +1,29 @@
+name: e2e_hardware
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
```

Comment:
Why different types specified?

Reply:
According to documentation https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request default types for pull_request event are: opened, synchronize, or reopened. Because I decide to redefine this array, I have put there all default events and new 'labeled'.